recipes/libftdi/1.x/conanfile.py

```diff
@@ -79,20 +80,27 @@ def build(self):
         cmake.build()
 
     def package(self):
-        copy(self, pattern="LICENSE", dst=os.path.join(self.package_folder, "licenses"), src=self.source_folder)
+        copy(self, "COPYING.LIB", self.source_folder, os.path.join(self.package_folder, "licenses"))
+        copy(self, "LICENSE", self.source_folder, os.path.join(self.package_folder, "licenses"))
+        if self.options.build_eeprom_tool or self.options.enable_cpp_wrapper:
+            copy(self, "COPYING.GPL", self.source_folder, os.path.join(self.package_folder, "licenses"))
         cmake = CMake(self)
         cmake.install()
-        lib_folder = os.path.join(self.package_folder, "lib",)
+        lib_folder = os.path.join(self.package_folder, "lib")
         rmdir(self, os.path.join(lib_folder, "cmake"))
         rmdir(self, os.path.join(lib_folder, "pkgconfig"))
 
     def package_info(self):
-        self.cpp_info.names['cmake_find_package'] = "LibFTDI1"
         self.cpp_info.names['cmake_find_package_multi'] = "LibFTDI1"
+        self.cpp_info.set_property("cmake_file_name", "LibFTDI1")
+        self.cpp_info.components["ftdi"].set_property("pkg_config_name", "libftdi1")
+        self.cpp_info.components["ftdi"].set_property("cmake_target_name", "LibFTDI1::ftdi")
```

Comment:
Will want this target name to be the default name since the upstream doesn't provide any official target names for us to use.

Reply:
@jwillikers Now the taget name and the module names are aligned. Thank you for your help.